msmbuilder/featurizer/featurizer.py

```diff
@@ -200,6 +200,61 @@ def partial_transform(self, traj):
         return x
 
 
+
+class StrucRMSDFeaturizer(Featurizer):
+    """Featurizer based on overall RMSD to a set of reference structures.
+
```

Comment:
Can you add the description that you included in the PR (about what this featurizer is computing) to the docstring. The first line is good, but you could add the rest as a more detailed description.